crates/config/src/config.rs

```diff
@@ -54,6 +54,9 @@ pub struct Config {
     pub p2p_network_config: Option<P2PNetworkConfig>,
     #[serde(default)]
     pub sync_server: SyncServerConfig,
+    /// Gasless tx support is enabled when this config presents.
+    #[serde(default)]
+    pub gasless_tx_support: Option<GaslessTxSupportConfig>,
```

Comment:
can we expose such config via RPC method `gw_get_node_info` https://github.com/godwokenrises/godwoken/blob/develop/crates/rpc-server/src/registry.rs#L1996 , I want to remove the duplicated config in godwoken-web3

Reply:
Done.